cli/src/commands/fix.rs

```diff
@@ -226,7 +226,16 @@ async fn fix_one_file(
                 // TODO: Because the stderr is passed through, this isn't always failing
                 // silently, but it should do something better will the exit code, tool
                 // name, etc.
-                Err(_) => prev_content,
+                Err(()) => {
+                    writeln!(
+                        ui.warning_default(),
+                        "Failed to run `{}` on `{}`",
+                        tool_config.command.split_name(),
+                        path_converter.format_file_path(&file_to_fix.repo_path)
+                    )
+                    .ok();
+                    prev_content
+                }
```

Comment:
I think it's slightly better to print error in `run_tool()` because we do that for non-zero exit.

A better option would be to send errors back to the main thread and print there, but doing that would require non-trivial changes.

Reply:
I moved it into `run_tool()`. That also meant that it was easier to distinguish between failure to start the tool and non-zero exit code, so I split this patch up into two for the two different cases.